resources/assets/wailaharvestability/lang/en_US.lang

```diff
@@ -30,3 +30,9 @@ option.harvestability.silktouchability=Silk Touchability
 option.harvestability.silktouchability.sneakingonly=Sneak only: Silk Touchability
 option.harvestability.harvestlevelnum=Harvest level no.
 option.harvestability.harvestlevelnum.sneakingonly=Sneak only: Harvest level no.
+
+# harvestlevels
+wailaharvestability.harvestlevel0=Wood
+wailaharvestability.harvestlevel1=Stone
+wailaharvestability.harvestlevel2=Iron
+wailaharvestability.harvestlevel3=Diamond
```

Comment:
Yes, but those are optional and only intended to be defined in resource packs distributed by modpacks. I don't want to have harvest level names hard-coded in Waila Harvestability, because what can be harvested for each level isn't always the same.